src/balatro_gym/cards/interfaces.py

```diff
@@ -136,10 +137,17 @@ def get_suit(self, card: "PlayingCard") -> Sequence[Suit]:
 
 
 class GlassCard(Enhancement):
+    _base_destruction_probability: float = 1 / 4
+
     def get_multiplication(self) -> float:
         # When scored
         return 2.0
 
+    def is_destroyed(self, probability_modifier: int = 1) -> bool:
```

Comment:
Suggestion: Instead of having a probability modifier, it might be easier to have a method that modifies the base probability when we have a joker that doubles all probabilities (and when we get rid of it)

Reply:
Cool. We can visit this again once support is added and/or there is a clear use case (like implementing Oops! All Sixes)